e2e/tests/questionTwo.test.ts

```diff
@@ -0,0 +1,8 @@
+import { test, expect } from '@playwright/test'
+const URL = 'http://localhost:5173/pokemon/2'
+test('Should be able to verify correct src of image for Ivysaur', async ({
+  page,
+}) => {
+  await page.goto(URL)
+  const Ivysaur = await page.getByRole('img').first()
```

Comment:
maybe add a test prompt here // WRITE YOUR TEST HERE

Reply:
Should we be specify what exactly needs to be tested?  Like, test filtering and sorting...

Reply:
added in a comment to write your test here